content/client-server-api/_index.md

```diff
@@ -1831,6 +1841,221 @@ the topic to be removed from the room.
+
+To allow the server to aggregate and find child events for a parent, the `m.relates_to`
+key of an event MUST be included in the plaintext copy of the event. It cannot be
+exclusively recorded in the encrypted payload as the server cannot decrypt the event
+for processing.
+
+{{% boxes/warning %}}
+If an encrypted event contains an `m.relates_to` in its payload, it should be
+ignored and instead favour the plaintext `m.relates_to` copy (including when there
+is no plaintext copy). This is to ensure the client's behaviour matches the server's
+capability to handle relationships.
+{{% /boxes/warning %}}
+
+Improperly formed or structured relationships are simply ignored. For example, the
+child and parent events being in different rooms or the relationship missing fields
+required by the schema below. The events would appear independent of each other or
+optionally with an error message (if rendered/handled by the client exclusively).
+
+`m.relates_to` is described as follows:
+
+{{% definition path="api/client-server/definitions/m.relates_to" %}}
+
+#### Relationship types
+
+{{% boxes/note %}}
+While this specification describes an `m.relates_to` object containing a `rel_type`, there
+is not currently any relationship type which uses this structure. Replies, described below,
+form their relationship outside of the `rel_type` in order to allow other, future, relationship
+types to make use of replies in addition to their normal behaviour.
+
+Custom `rel_type`s can, and should, still use the schema described above for relevant
+behaviour.
+{{% /boxes/note %}}
+
+This specification describes the following relationship types:
+
+* [Rich replies](#rich-replies) (**Note**: does not use `rel_type`).
+
+{{% boxes/note %}}
+This specification does not currently define any relation type which requires
+aggregation or has restrictions, however [namespaced](/appendices#identifier-grammar)
+relationship types might have these restrictions.
+
+Future versions of this specification are expected to require certain behaviours
+or aggregation of related events.
+{{% /boxes/note %}}
+
+#### Aggregations
+
+{{% added-in v="1.3" %}}
+
+Some child events can be "aggregated" or "bundled" by the server, depending on their
+`rel_type`. This can allow a set of child events to be summarised to the client without
+the client needing the child events themselves.
+
+An example of this might be that a `rel_type` requires an extra `key` field which, when
+appropriately specified, would mean that the client receives a total count for the number
+of times that `key` was used by child events.
+
+The actual aggregation format depends on the `rel_type`.
+
+{{% boxes/note %}}
+This specification does not currently describe any `rel_type` which require
+aggregation, however [namespaced](/appendices#identifier-grammar) relationship
+types might have aggregation behaviour.
+{{% /boxes/note %}}
+
+When aggregations are summarised on an event, it is known as a "bundled aggregation"
```

Comment:
😬 I thought it was the same thing - will take a look and see what I can do about it.

Reply:
aggregation is the process of building up the summary of relating events. You can get hold of that aggregation in various ways; one of them is by having them *bundled* into said event.  That's AIUI, anyway.

Reply:
I don't think there's any other way o get hold of aggregations anymore -- the `/aggregations` endpoint was removed from MSC2675. That makes them essentially the same thing.

Reply:
Ah, that's useful context, thanks @clokep.

Still, I find it easier to think about them as slightly separate concepts - not least because it's conceivable we might reintroduce another way of getting the aggregations in future.

Reply:
> Still, I find it easier to think about them as slightly separate concepts - not least because it's conceivable we might reintroduce another way of getting the aggregations in future.

Yeah, that's fair. I assume your definitions are something like:

* Aggregation: a format for returning a summary or grouping of child events which share a parent event.
* Bundled aggregations: pre-calculated aggregations which are returned to clients as part of the `unsigned` field by the server.